server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -388,32 +357,74 @@ private QueryDataSet processShowStorageGroup() {
     return listDataSet;
   }
 
+  private QueryDataSet processShowTimeseriesWithIndex(ShowTimeSeriesPlan timeSeriesPlan) throws MetadataException {
+    List<ShowTimeSeriesResult> timeseriesList = MManager.getInstance()
+            .getAllMeasurementSchema(timeSeriesPlan.getPath().toString(),
+                    timeSeriesPlan.isContains(), timeSeriesPlan.getKey(), timeSeriesPlan.getValue());
```

Comment:
Should we just pass `timeSeriesPlan` as param of getAllMeasurementSchema()  method? As `createTimeseries(createTimeSeriesPlan)` did.

Reply:
fixed